package.json

```diff
@@ -54,25 +54,22 @@
     "karma-jasmine": "~4.0.0",
     "chromium": "^3.0.3",
     "puppeteer": "^15.5.0",
-    "ng-packagr": "^14.0.2",
+    "ng-packagr": "^15.2.2",
     "protractor": "~7.0.0",
     "rxjs": "~6.6.0",
     "ts-node": "~8.3.0",
-    "tslib": "^2.0.0",
-    "typescript": "~4.7.4",
+    "tslib": "^2.3.0",
+    "typescript": "~4.9.5",
     "zone.js": "~0.11.4"
   },
   "dependencies": {
     "@microsoft/applicationinsights-shims": "^2.0.2",
     "@microsoft/applicationinsights-common": "^2.8.12",
     "@microsoft/applicationinsights-core-js": "^2.8.12",
-    "@microsoft/applicationinsights-properties-js": "^2.8.12",
-    "tslib": "^2.0.0"
+    "@microsoft/applicationinsights-properties-js": "^2.8.12"
   },
   "peerDependencies": {
-    "@microsoft/applicationinsights-analytics-js": "^2.8.12",
-    "@angular/common": ">= 14.0.3",
```

Comment:
Without these it would require that projects add their own dependencies and this could lead to reported issues about missing peer dependencies. If this is compatible with 14 then I think these should remain.

Reply:
I'm not sure what you mean by "would require that projects add their own dependencies" since this is the root package.json file.

Reply:
Same response as above, dev === build time only and not using